client/src/pages/Home.js

```diff
@@ -17,7 +18,11 @@ export function Home() {
 				setBookings(data);
 			})
 			.catch((err) => console.log(err));
-	}, []);
+	}, [refreshKey]);
+
+	const refreshBooking = () => {
+		setRefreshKey((oldKey) => oldKey + 1);
```

Comment:
I like that you are updating the state in here the right way!
Without directly mutating the state ;)

Reply:
Thanks @teomrd